src/App.tsx

```diff
@@ -1,10 +1,51 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 11361;
+
+// https://mate.academy/students-api/todos?userId=11361
+
+// const todos = client.get('/todos?userId=11361');
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [updatedTodos, setuptadedTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState(false);
+  const [sortBy, setSortBy] = useState<string>('all');
+
+  useEffect(() => {
+    client.get<Todo[]>('/todos?userId=11361')
+      .then((json: Todo[]) => setTodos(json))
+      .catch(() => {
+        setError(true);
+        setTimeout(() => setError(false), 3000);
+      });
+  }, []);
+
+  useEffect(() => {
```

Comment:
think this useEffect is redundant here, just update setUpdatedTodos in 1st useEffect when fetch all todos, I added this there already for suggestion